app/build.gradle

```diff
@@ -2,11 +2,12 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-	namespace 'de.markusfisch.android.binaryeye'
-	compileSdkVersion sdk_version
 	defaultConfig {
-		minSdkVersion 9
-		targetSdkVersion sdk_version
+		applicationId 'de.markusfisch.android.binaryeye'
```

Comment:
This is how it is done in practically all Android apps, I also don't see any harm in keeping it.

Reply:
Well, it's in practically all Android apps because Android Studio creates new projects with it, and probably because nobody knows it's optional.

Now, I'm not trying to convince you if you like to specify it. I'm just trying to make clear that I like to limit things to what is necessary. For my taste, I like it when you can't leave anything out - and not when you can't add anything in.

So I do consider unnecessary redundancies to be at least a potential problem. Now that every app must contain a `namespace`, and this should reasonably always correspond to the `applicationId`, duplicating this String definitely feels wrong to me. Whenever I need a string in two different places, I use a symbol. Of course, you could do the same here - but what for, if `applicationId` is not necessary anyway?

Sorry for being picky about these (completely unimportant) things. You don't need to take this out, but I will likely remove it after the merge. Just so you know 😉

Reply:
Thanks for the detailed explanation and letting me know that it is optional. I totally understand your reasoning and I respect people's different coding styles. If you are going to remove it after the merge anyway, I see no reason on pushing it in the first place. I have now removed the `applicationId` from this PR.